terraform.tf

```diff
@@ -10,5 +10,9 @@ terraform {
       source  = "hashicorp/random"
       version = ">= 3.5.0"
```

Comment:
Recommended to update to something like below as per spec TFNFR26
version = ">= 3.5.0, < 4.0.0"